components/Icon/services/icons.js

```diff
@@ -0,0 +1,57 @@
+import _ from "lodash";
+import { defaultConfig } from "../assets";
+let Icons = {};
+
+registerIcons(defaultConfig);
+
+export function registerIcons(config) {
+  const isArray = _.isArray(config);
+  const isObject = _.isObject(config);
+
+  if (!isObject) {
+    return console.warn(
+      "Icon config must be an object or an array of objects."
+    );
+  }
+
+  if (isArray) {
+    return _.forEach(config, function (element) {
```

Comment:
I don't like the return inconsistency here. 
Currently we have returns for console.warn OR set of icons OR nothing (the 'else' part of this if block).

Also, can we use arrow function here?